examples/react-native/src/features/InAppMessaging/Demo/Example.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import { Amplify } from 'aws-amplify';
 import { SafeAreaView, StyleSheet, Text, View } from 'react-native';
 
 import { withInAppMessaging } from '@aws-amplify/ui-react-native';
 import { Button, Checkbox, Radio, RadioGroup } from '../../../ui';
 import { useInAppDemo, ACTIONS, LAYOUTS, ORIENTATIONS } from './utils';
+import config from './aws-exports';
+
+Amplify.configure(config);
```

Comment:
Added call to `Amplify.configure` here because InApp now requires `initializeInAppMessaging()` to use, which must be called with `Amplify.configure`